app/src/main/java/org/openobservatory/ooniprobe/domain/GenerateAutoRunServiceSuite.java

```diff
@@ -77,7 +78,8 @@ public boolean shouldStart(Boolean isWifi, Boolean isCharging, Boolean isVPNInUs
             return false;
         if (pm.testChargingOnly() && !isCharging)
             return false;
-
+        if(ReachabilityManager.getChargingLevel(app) < 20)
```

Comment:
Should we perhaps still run the test if the phone is charging irrespective of the battery level?

Reply:
Fixed